app/controllers/nodes_controller.rb

```diff
@@ -16,6 +18,20 @@ def show
     @minion = Minion.find(params[:id])
   end
 
+  def bootstrap
+    available_roles = %i(master)
+    Minion.where(role: nil).each do |minion|
```

Comment:
Are we going to have this big number of minions when bootstrapping? `find_each` is more focused for batched finds (`find_in_batches`). The default batch is `1000` that I doubt we will reach on this result, so they are almost equivalent, but it's ok, we can use `find_each`.